west.yml

```diff
@@ -108,7 +108,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: v1.9.0
+      revision: pull/663/head
```

Comment:
Please update SHA: `43c662addc3e89f7fd36a9733f6a34b5ef16b928` (and rebase).

Reply:
Done